test/Microsoft.AspNetCore.Server.KestrelTests/KestrelServerTests.cs

```diff
@@ -75,6 +75,14 @@ public void StartWithMaxRequestBufferSizeLessThanMaxRequestLineSizeThrows(long m
             Assert.Equal(1, testLogger.CriticalErrorsLogged);
         }
 
+        [Fact]
+        public void LoggerCategoryNameIsKestrelServerNamespace()
+        {
+            var mockLoggerFactory = new Mock<ILoggerFactory>();
+            new KestrelServer(Options.Create<KestrelServerOptions>(null), new LifetimeNotImplemented(), mockLoggerFactory.Object);
+            mockLoggerFactory.Verify(factory => factory.CreateLogger(typeof(KestrelServer).GetTypeInfo().Namespace));
```

Comment:
If we really want to test we don't break things by changing the category name, shouldn't we just have the namespace as the string literal "Microsoft.AspNetCore.Server.Kestrel". I don't think that anyone will change the namespace of KestrelServer anytime soon since it's public, but might as well be sure.